app/views/state_file/questions/eligible/edit.html.erb

```diff
@@ -28,7 +28,15 @@
         </ul>
       <% elsif current_state_code == "nc" %>
         <p>Copy needed...</p>
-      <% end %>
+    <% elsif current_state_code == "id" %>
+      <ul class="list--bulleted">
+        <li><%= t(".child_care_deduction") %></li>
+        <li><%= t(".interest_from_obligations") %></li>
+        <li><%= t(".social_security_retirement_deduction") %></li>
+        <li><%= t(".id_child_tax_credit") %></li>
+        <li><a href=" https://tax.idaho.gov/taxes/income-tax/individual-income/popular-credits-and-deductions/idaho-grocery-credit" target="_blank" rel="noopener nofollow"><%= t(".id_grocery_credit") %></a></li>
+      </ul>
+    <% end %>
```

Comment:
i wonder if it's worth pulling these into state-specific partials too like `"state_file/questions/eligible/#{current_state_code}_deductions_and_credits"` like how it's implemented for `_unsupported` partials below